src/enrichment_facility_tests.cc

```diff
@@ -259,175 +515,10 @@ TEST_F(EnrichmentFacilityTest, ValidReq) {
-TEST_F(EnrichmentFacilityTest, Accept) {
-  using cyclus::Bid;
-  using cyclus::Material;
-  using cyclus::Request;
-  using cyclus::Trade;
-
-  // an enrichment facility gets two trades, each for 1/3 of its inv size
-  // note that comp != recipe is covered by AddMat tests
-  // note that qty >= inv capacity is covered by toolkit::ResourceBuff tests
-
-  double qty = inv_size / 3;
-  std::vector< std::pair<cyclus::Trade<cyclus::Material>,
-                         cyclus::Material::Ptr> > responses;
-
-  Request<Material>* req1 =
-      Request<Material>::Create(DoRequest(), src_facility, in_commod);
-  Bid<Material>* bid1 =
-      Bid<Material>::Create(req1, GetMat(qty), trader);
-
-  Request<Material>* req2 =
-      Request<Material>::Create(DoRequest(), src_facility, in_commod);
-  Bid<Material>* bid2 = Bid<Material>::Create(req2, GetMat(qty), trader);
-
-  Trade<Material> trade1(req1, bid1, qty);
-  responses.push_back(std::make_pair(trade1, GetMat(qty)));
-  Trade<Material> trade2(req2, bid2, qty);
-  responses.push_back(std::make_pair(trade2, GetMat(qty)));
-
-  EXPECT_DOUBLE_EQ(0.0, src_facility->InventorySize());
-  src_facility->AcceptMatlTrades(responses);
-  EXPECT_DOUBLE_EQ(qty * 2, src_facility->InventorySize());
-
-  delete bid2;
-  delete bid1;
-  delete req2;
-  delete req1;
-}
-
-// - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
-TEST_F(EnrichmentFacilityTest, AddBids) {
-  using cyclus::Bid;
-  using cyclus::BidPortfolio;
-  using cyclus::CapacityConstraint;
-  using cyclus::Converter;
-  using cyclus::ExchangeContext;
-  using cyclus::Material;
-  cyclus::Env::SetNucDataPath();
-  // an enrichment facility bids on nreqs requests
-  // note that bid response is covered by Bid tests
-  // note that validity of requests is covered by ValidReq tests
-  int nreqs = 5;
-  int nvalid = 4;
-
-  // set up inventory
-  double current_size = inv_size / 2;  // test something other than max size
-  DoAddMat(GetMat(current_size));
-
-  boost::shared_ptr< cyclus::ExchangeContext<Material> >
-      ec = GetContext(nreqs, nvalid);
-
-  std::set<BidPortfolio<Material>::Ptr> ports =
-      src_facility->GetMatlBids(ec.get()->commod_requests);
-
-  ASSERT_TRUE(ports.size() > 0);
-  EXPECT_EQ(ports.size(), 1);
-
-  BidPortfolio<Material>::Ptr port = *ports.begin();
-  EXPECT_EQ(port->bidder(), src_facility);
-  EXPECT_EQ(port->bids().size(), nvalid);
-
-  const std::set< CapacityConstraint<Material> >& constrs = port->constraints();
-  Converter<Material>::Ptr sc(new SWUConverter(feed_assay, tails_assay));
-  Converter<Material>::Ptr nc(new NatUConverter(feed_assay, tails_assay));
-  CapacityConstraint<Material> swu(swu_capacity, sc);
-  CapacityConstraint<Material> natu(current_size, nc);
-  EXPECT_EQ(constrs.size(), 2);
-  EXPECT_TRUE(*constrs.begin() == swu || *(++constrs.begin()) == swu);
-  EXPECT_TRUE(*constrs.begin() == natu || *(++constrs.begin()) == natu);
-}
-
```

Comment:
Tested that constraints were properly added to EF bids - implicitly tested by MockSim in the new tests CheckSWUConstraint and CheckCapConstraint